docs/UserGuide.md

```diff
@@ -988,18 +1108,19 @@ You may want to **re-enter your client data in a fresh JSON file** in the event
+| `r/Remark`                                                            | <ul><li>**Can be empty** if remark is not specified.</li><li>Must not include other command prefixes (`a/`,`b/`,`e/`,`f/`,`h/`,`i/`,`n/`,`p/`,`r/`,`t/`) to prevent parsing errors. For instance, `a/lemontree street t/1` may cause the command to fail, as the system will interpret `t/` as an unintended tag prefix.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | <ul><li>✅ <code>r/Likes to eat cake</code></li></ul>                                                                  |
+| `b/Birthday`                                                          | <ul><li>BIRTHDAY should be in the form "DDMMMYYYY", and can be empty if the birthday is not specified. <li>The date must not be in the future.</li><li>The date must exist in the Gregorian calendar. (<code>b/29Feb2023</code> is not allowed as it is not a valid day to begin with)</li><li>The day "DD" must be numeric. For 1st-9th day of the month, the 0 need not be present. </li><li>The month "MMM" refers to the first 3 letters of the month (case-insensitive)</li><li>The year "YYYY" must be in full and greater than or equal to 1000.</li></ul>                                                                                                                                                                                                                                                                                                     | <ul><li>✅ <code>b/2Feb2002</code></li><li>✅ <code>b/23Apr1972</code></li><li>❌ <code>b/23Apr72</code></li></ul> </ul> |
+
 
```

Comment:
I cant add comment below this but anyways for

Command Summary
**Help (individual commands)** i think best to link to the actual help command as they may again raise the flaw of why some commands are not supported

Reply:
actually the only one that is not covered is exit

Reply:
i made the change:

* For the full list of commands, you can use the general [help command](#help-help)